.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We wont need copyright on this file and also the format for a comment on a markdown is similar to `html`.
https://alvinalexander.com/technology/markdown-comments-syntax-not-in-generated-

- we would want the description to be the first thing in the PR template (as is in ai4web). 
- in last engineering sync, we decided to get rid of `notes for reviewers` section as no one was using it. Lets be consistent with other projects that we own.

Reply:
Accepted.